src/lib.rs

```diff
@@ -29,6 +29,10 @@ impl Message {
         String::from_utf8_lossy(&self.value).to_string()
     }
 
+    pub fn get_value(&self) -> &Vec<u8> {
+        &self.value
```

Comment:
Why not make value public?

Reply:
I've followed the same idea of `get_label`